### PR TITLE
Patch definition for 6.45

### PR DESCRIPTION
**Pull request by AkairyuGestalter**
Since we actually have BLU analysis now (yay!), we're going to need to be able to split between pre-6.45 level 70 cap spell expectations and 6.45+ level 80 cap spells. PR adds 6.45 to the patch list and creates empty action and status layer files

### CODE REVIEW

**Comment by AkairyuGestalter** [comment]
One new UnableToAct status added by yarn generate. Went ahead and saved other job maintainers some busywork by updating job support patch levels, since only BLU has job changes this patch.

File: src/parser/jobs/sch/index.js
Comment (nonowazu) [inline src/parser/jobs/sch/index.js:21]: I know you don't need my explicit approval, but I :+1: this

**Review by nonowazu — APPROVED** [review]

**Review by ackwell — APPROVED** [review]

**Pull request merged**
